nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-hadoop-record-utils/src/main/java/org/apache/nifi/processors/hadoop/AbstractPutHDFSRecord.java

```diff
@@ -279,8 +279,18 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 createDirectory(fileSystem, directoryPath, remoteOwner, remoteGroup);
 
                 // write to tempFile first and on success rename to destFile
-                final Path tempFile = new Path(directoryPath, "." + filenameValue);
-                final Path destFile = new Path(directoryPath, filenameValue);
+                final Path tempFile = new Path(directoryPath, "." + filenameValue) {
```

Comment:
This looks like an unrelated change, should it be reverted?

Reply:
Yes I think I had another PR's changes in mine at the same time, will check. Great catch!